api/src/db/migrations/v73_add_operating_status_field.ts

```diff
@@ -0,0 +1,324 @@
+import { formationTaskId } from "@shared/gradualGraduationStages";
+import { v72UserData } from "./v72_add_real_estate_management";
+
+export interface v73UserData {
+  user: v73BusinessUser;
+  profileData: v73ProfileData;
+  formProgress: v73FormProgress;
+  taskProgress: Record<string, v73TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v73LicenseData | undefined;
+  preferences: v73Preferences;
+  taxFilingData: v73TaxFilingData;
+  formationData: v73FormationData;
+  version: number;
+}
+
+export const migrate_v72_to_v73 = (v72Data: v72UserData): v73UserData => {
+  const taskProgress = v72Data.taskProgress;
+
+  let newOperatingPhase: v73OperatingPhase;
+  if (
+    v72Data.profileData.initialOnboardingFlow === "OWNING" ||
+    v72Data.profileData.businessPersona === "OWNING"
+  ) {
+    newOperatingPhase = "UP_AND_RUNNING";
+  } else if (v72Data.taskProgress["register-for-taxes"] === "COMPLETED") {
+    newOperatingPhase = "FORMED_AND_REGISTERED";
+  } else if (v72Data.taskProgress[formationTaskId] === "COMPLETED") {
+    newOperatingPhase = "NEEDS_TO_REGISTER_FOR_TAXES";
```

Comment:
is this correct for the distinction based on legal structures? An SP/GP who has not completed taxes should be NEEDS_TO_REGISTER right?